docs/account-experience/index.mdx

```diff
@@ -4,6 +4,12 @@ title: Account Experience Overview
 sidebar_label: Account Experience
 ---
 
+import CanonicalUrl from "@site/src/components/CanonicalUrl"
+
+<CanonicalUrl path="/account-experience" />
+
+---
```

Comment:
This `<hr>` does not belong here.

<img width="3058" height="904" alt="Image" src="https://github.com/user-attachments/assets/cbada071-e079-48e6-8caf-b5eb97f10424" />